cypress/e2e/tests/accessibility/shell.spec.ts

```diff
@@ -352,29 +351,12 @@ describe('Shell a11y testing', { tags: ['@adminUser', '@accessibility'] }, () =>
       });
 
       describe('Workloads', () => {
-        it('Workloads page', () => {
-          const workloadsListPage = new WorkloadsListPageBasePo('local', 'workload');
-
-          WorkloadsListPageBasePo.navTo();
-          workloadsListPage.waitForPage();
-          workloadsListPage.sortableTable().checkLoadingIndicatorNotVisible();
-          // expand the health scale up/down control
-          workloadsListPage.details('rancher', 8).should('be.visible');
-          workloadsListPage.details('rancher', 8).click();
-
-          cy.injectAxe();
-
-          cy.checkPageAccessibility();
-        });
-
         it('Deployments page', () => {
           const deploymentsListPage = new WorkloadsDeploymentsListPagePo();
 
           deploymentsListPage.goTo();
           deploymentsListPage.waitForPage();
           deploymentsListPage.sortableTable().checkLoadingIndicatorNotVisible();
-          // expand the health scale up/down control
-          deploymentsListPage.sortableTable().getTableCell(1, 10).click();
```

Comment:
make alex and phil aware that workloads page/list has been removed. let a11y company aware